packages/core/core.ts

```diff
@@ -42,20 +41,23 @@ export function registerGlobals() {
 	globalThis.HookManager = hooking.hookManager;
 }
 
-export function initFuzzing(options: Options) {
+export async function initFuzzing(options: Options) {
 	registerGlobals();
 
-	options.customHooks.forEach((customHook) => {
-		importModule(customHook);
-	});
+	const importCustomHookPromises = options.customHooks.map((customHook) =>
```

Comment:
This could be reduced to `await Promise.all(options.customHooks.map(importModule));`

Reply:
Simplified the statement.